app/Http/Controllers/RoleController.php

```diff
@@ -8,6 +8,9 @@
 
 class RoleController extends Controller
 {
+
+    private string $redirectTo = 'roles.index';
```

Comment:
Not a fan of this refactor. I understand the logic, but its inpractical in laravel.

@alexstojda can we make the repeated string check higher or mark it as a false positive?

Reply:
@alecadub You can mark it as a false positive in Sonarcloud and undo the change